src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -3617,6 +3617,30 @@
     </Field>
     <Field Name="EqualsValue" Type="EqualsValueClauseSyntax" Optional="true"/>
   </Node>
+  <Node Name="ExtensionDeclarationSyntax" Base="TypeDeclarationSyntax">
+    <TypeComment>
+      <summary>Extension container syntax.</summary>
+    </TypeComment>
+    <Kind Name="ExtensionDeclaration" />
+    <Field Name="AttributeLists" Type="SyntaxList&lt;AttributeListSyntax&gt;" Override="true"/>
+    <Field Name="Modifiers" Type="SyntaxList&lt;SyntaxToken&gt;" Override="true"/>
+    <Field Name="Keyword" Type="SyntaxToken" Override="true">
+      <ContextualKind Name="ExtensionKeyword"/>
```

Comment:
i'm actually slightly surprised by this.  i would have thought this would be `<kind`